Code/Framework/AzFramework/AzFramework/FileFunc/FileFunc.cpp

```diff
@@ -126,9 +126,9 @@ namespace AzFramework
             *      1. Key exists, value is identical -> do nothing
             *          This can occur if a user modifies a config file while the tooling is open.
             *          Example:
-            *              1) Load Project Configurator.
+            *              1) Load Project Manager.
             *              2) Modify your bootstrap.cfg to a different project.
-            *              3) Tell Project Configurator to set your project to the project you set in #2.
+            *              3) Tell Project Manager to set your project to the project you set in #2.
```

Comment:
Project Manager doesn't edit bootstrap.cfg or other cfg files

Reply:
This method actually is no longer used, so will just delete it